### PR TITLE
remove repetitive words

### PR DESCRIPTION
**Pull request by pengqiseven**

### CODE REVIEW

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
This affects the swagger spec, please run `make generate-swagger`.

**Review by techknowlogick — APPROVED** [review]

File: templates/swagger/v1_json.tmpl [hunk 1]
```diff
@@ -24821,4 +24821,4 @@
       "TOTPHeader": []
     }
   ]
-}
+}
```
Comment (harryzcy) [inline templates/swagger/v1_json.tmpl]: Please add back the newline here. Related lint error:

```
templates/swagger/v1_json.tmpl:
	Wrong line endings or no final newline
```

**Review by harryzcy — COMMENTED** [review]
Lint error is related

File: templates/swagger/v1_json.tmpl (hunk 1 above)
Comment (pengqiseven) [reply templates/swagger/v1_json.tmpl]: Modified！

Please review again

**Review by silverwind — APPROVED** [review]

**Pull request merged**